pyopenms_viz/_matplotlib/core.py

```
@@ -228,7 +228,9 @@ def _add_bounding_vertical_drawer(self):
             "Matplotlib does not support interactive plots and cannot use method '_add_bounding_vertical_drawer'"
         )
 
-    def generate(self, tooltips, custom_hover_data) -> Axes:
+    def generate(
+        self, tooltips, custom_hover_data, _fixed_tooltip_for_trace=True
```

Ruff (0.14.13)

Unused method argument: `tooltips`

(ARG002)